src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,26 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File file = new File(fromFileName);
+        try {
+            List<String> strings = Files.readAllLines(file.toPath());
+            System.out.println(strings);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read fails ", e);
+        }
+        try {
+            Files.write(file.toPath(), toFileName.getBytes(), StandardOpenOption.APPEND);
```

Comment:
The method writes to the same file it reads from (`fromFileName`). If the intention is to write to a different file, you should use `toFileName` as the path in `Files.write()` instead of `file.toPath()`.

Reply:
how do it?